pyiron_base/table/datamining.py

```diff
@@ -692,8 +690,7 @@ def from_hdf(self, hdf=None, group_name=None):
         if hdf_version=="0.3.0":
             with self.project_hdf5.open("output") as hdf5_output:
                 if "table" in hdf5_output.list_groups():
-                    data = hdf5_output["table"].to_object().to_builtin()
-                    self._pyiron_table._df = pandas.DataFrame(data)
+                    self._pyiron_table._df = pandas.read_hdf(hdf5_output.file_name, hdf5_output.h5_path + "/table")
```

Comment:
This will break without `pytables` or not?

Reply:
Yes, I rely on pytables - but as long as that is the recommended solution by pandas I guess that is the only option. Alternatively we could also pickle the object and store the pickled string, but I do not think that is a better alternative.

Reply:
Then there's no need to put the `DataFrame` into the data container at all and we can just `DataFrame.to_hdf`/`read_hdf`, I suppose.  Then we also need to put `tables` back into our `environment.yml`.

Reply:
That I exactly what I do in the latest commit.

Reply:
Ah, I missed that.  Looks good to me then, we can always come back to this if someone really wants to store "complex" data in their pyiron tables.